build.gradle

```diff
@@ -131,10 +128,10 @@ allprojects {
   }
 
   repositories {
+    mavenLocal()
+    mavenCentral()
     jcenter()
```

Comment:
still needed for one old spring lib will deal with it later:
https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020